### DISCUSSION TITLE
change Essentials config home section

### DISCUSSION THREAD

**Discussion by kretoz** [EssentialsX/Essentials#4906] [Ideas and Feature Suggestions] [answered]
hello in Ess config home section when i define multiply homes, for key i cant put a numbers
can u chnage the key requerment from a String to String or Numbers
i am making an skript where i can add more homes via permission. 
So basically i need like this essentials.sethome.multiple.4
to use the permission integer to add more homes, otherwise i cant

**Comment by mdcfe** [comment, marked as answer]
You need to wrap number keys in quotes, ie `"4": 4`.
